lib/active_record/connection_adapters/oracle_enhanced_adapter.rb

```diff
@@ -491,7 +491,7 @@ def prefetch_primary_key?(table_name = nil)
         do_not_prefetch = @do_not_prefetch_primary_key[table_name]
         if do_not_prefetch.nil?
           owner, desc_table_name = @connection.describe(table_name)
-          @do_not_prefetch_primary_key [table_name] = do_not_prefetch = !has_primary_key?(table_name, owner, desc_table_name)
+          @do_not_prefetch_primary_key[table_name] = do_not_prefetch = !has_primary_key?(table_name, owner, desc_table_name)
```

Comment:
Would you rebase this pull request based on the current master branch? because https://github.com/rsim/oracle-enhanced/pull/2108 addresses this unnecessary spece.